contracts/governance/Governor.sol

```diff
@@ -28,9 +28,20 @@ import {IGovernor, IERC6372} from "./IGovernor.sol";
 abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receiver, IERC1155Receiver {
     using DoubleEndedQueue for DoubleEndedQueue.Bytes32Deque;
 
-    bytes32 public constant BALLOT_TYPEHASH = keccak256("Ballot(uint256 proposalId,uint8 support)");
+    bytes32 public constant BALLOT_TYPEHASH =
+        keccak256(
+            "Ballot("
+            "uint256 proposalId,"
+            "uint8 support)"
```

Comment:
I'm not sure if I like this or if it would be better to put the parenthesis on its own line...

```solidity
            "Ballot("
            "uint256 proposalId,"
            "uint8 support"
            ")"
```

What do you think?

Reply:
I found it more compact while not giving up on readability